libkbfs/mdserver_remote.go

```diff
@@ -622,13 +631,28 @@ func (md *MDServerRemote) GetForTLF(ctx context.Context, id tlf.ID,
 		LockBeforeGet: lockBeforeGet,
 	}
 
-	_, rmdses, err := md.get(ctx, arg)
-	if err != nil {
+	_, rmdses, err := md.get(getCtx, arg)
+	switch errors.Cause(err) {
+	case nil:
+	case context.DeadlineExceeded:
+		if cachedRmds != nil {
+			md.log.CDebugf(ctx, "Can't contact server; using cached MD")
+			return cachedRmds, nil
+		}
+		return nil, err
+	default:
```

Comment:
In offline mode, we might get something like "network unreachable" here right? Do we need to capture those networking errors here?

Reply:
Ah that's right! I'm having a Deja Vu for this question and answer. Sorry for my brain fart.